incidenter/program.py

```diff
@@ -0,0 +1,145 @@
+from os import getenv
+import re
+
+from autokitteh import next_event, subscribe
+from autokitteh.slack import slack_client
+
+from height import add_task_message
+from height import create_task
+from zoom import create_meeting
+
+
+CHANNEL_PREFIX = getenv("CHANNEL_PREFIX")
+
+slack_client = slack_client("slack")
+
+pattern = re.compile(r"<.+> incident *(.*)")
+
+
+def on_slack_app_mention(event):
+    m = pattern.match(event.data.text)
+    if not m:
```

Comment:
It seems like a better idea to put this regex filter in the trigger, but it's enough to put a TODO reminder if you don't want to do it now

Reply:
this makes sense... will check